doc/user: document configurable group and transactional IDs for Kafka (#23793)
MaterializeInc/materialize: doc/user/content/sql/system-catalog/mz_internal.md

| Field                  | Type           | Meaning                                                                                                   |
|------------------------|----------------|-----------------------------------------------------------------------------------------------------------|
| `id`                   | [`text`]       | The ID of the Kafka source. Corresponds to [`mz_catalog.mz_sources.id`](../mz_catalog#mz_sources).        |

> | `group_id_prefix`      | [`text`]       | The prefix of the group ID that Materialize will use when consuming data for the Kafka source.            |

Reviewer: ```suggestion
| `group_id_prefix`      | [`text`]       | The prefix of the group ID to use when consuming data for the Kafka source.            |
```

Reviewer: It isn't very useful, but I hate inconsistency more!
